drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-backend/pom.xml

```diff
@@ -133,6 +133,26 @@
       <artifactId>xstream</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.drools</groupId>
+      <artifactId>drools-wb-dtable-xls-editor-api</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>org.apache.poi</groupId>
+      <artifactId>poi</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.poi</groupId>
+      <artifactId>poi-ooxml</artifactId>
```

Comment:
`mvn dependecy:analyze` marks this as declared unused dependency, was this added due to CDI tests? if that is the case, could we please add a comment about it?

Reply:
This gives XLSX support that I never ended up using. I'll remove it.